<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SPRulePatternQuickInputDialog</class>
 <widget class="QDialog" name="SPRulePatternQuickInputDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>964</width>
    <height>622</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <widget class="QSplitter" name="horizontalSplitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QSplitter" name="verticalSplitter">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QGroupBox" name="exampleTextGroupBox">
       <property name="title">
        <string>Step 1: Input example text</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="TextEditor" name="textEdit"/>
        </item>
       </layout>
      </widget>
      <widget class="QGroupBox" name="specialBlockGroupBox">
       <property name="minimumSize">
        <size>
         <width>440</width>
         <height>0</height>
        </size>
       </property>
       <property name="title">
        <string>Step 2: Mark special text blocks</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QTableView" name="specialBlockTableView"/>
        </item>
        <item>
         <widget class="QCheckBox" name="autoMarkSpecialBlockCheckBox">
          <property name="text">
           <string>Auto mark recognized special text blocks</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QGroupBox" name="previewGroupBox">
      <property name="title">
       <string>Step 3: Preview and fine-tune</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="SPRulePatternInputWidget" name="rulePreviewWidget" native="true"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TextEditor</class>
   <extends>QPlainTextEdit</extends>
   <header>src/gui/TextEditor.h</header>
  </customwidget>
  <customwidget>
   <class>SPRulePatternInputWidget</class>
   <extends>QWidget</extends>
   <header>src/gui/SimpleParser/SPRulePatternInputWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>SPRulePatternQuickInputDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>SPRulePatternQuickInputDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
